src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpMultilineCheck.java

```diff
@@ -73,15 +73,48 @@
  * </li>
  * </ul>
  * <p>
+ * To run the check with its default configuration:
```

Comment:
`To run the check with its default configuration (no matches will be):`

Reply:
@stryder2000 , please do not resolve my review points.
I will do this myself as confirmation that I am ok with fix.

Reply:
Okay @romani will make sure next time 👍